infrastructure/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "infrastructure",
+  "version": "0.1.0",
+  "bin": {
+    "infrastructure": "bin/infrastructure.js"
+  },
+  "scripts": {
+    "build": "tsc",
```

Comment:
I wonder if we should compile this with jsii so it we will "protect" us from using any unsupported TS features